targets/ChibiOS/_WiFi/inventek/sockets_ism43362.cpp

```
@@ -0,0 +1,887 @@
+    return SOCK_SOCKET_ERROR;
+}
+
+int SOCK_shutdown(SOCK_SOCKET socket, int how)
+{
+    (void)how;
+
+    // ES-WIFI doesn't support half-close, so shutdown behaves the same as close
+    return SOCK_close(socket);
+}
+
+int SOCK_getaddrinfo(
```

Cppcheck (2.21.0)

[style] The function 'SOCK_getaddrinfo' is never used.

(unusedFunction)